dali/test/python/test_operator_warp.py

```diff
@@ -63,9 +63,9 @@ def warp_fixed(img):
   return warp_fn
 
 
-class NewWarpPipeline(Pipeline):
+class WarpPipeline(Pipeline):
```

Comment:
could you run (locally is enough) a simple test that compares `WarpPipeline` to `NewWarpPipeline` (use `compare_pipelines` for that)

Reply:
There's a test in test_pipeline.py which compares against OpenCV and it still passes after the switch - I only needed to remove the `use_image_center` flag, which now is not supported (it makes little sense for  WarpAffine).